<launch>
  <node name="camera_wizard" pkg="theora_webcams" type="camera_publisher" output="screen"/>
</launch>
